sourceclear.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
Not a fan of top level sourceclear.sh file. Please create a scripts folder and throw it in there.

Reply:
+1. Same model is used in other SDKs see https://github.com/optimizely/objective-c-sdk/tree/master/Scripts